.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow will build a Java project with Maven
```

Comment:
should the comments be adapted?

Reply:
its just a comment you can simply remove it or adapt it for the ui